tile_generator/package_flags.py

```diff
@@ -263,3 +266,58 @@ def _apply(self, config_obj, package, release):
             'buildpack_order': '(( .properties.{}_buildpack_order.value ))'.format(packagename),
         })
         package['properties'] = properties
+
+class Helm(FlagBase):
+    @classmethod
+    def _apply(self, config_obj, package, release):
+        # Read the helm chart and bundle all required docker images
+        chart_info = helm.get_chart_info(package['path'])
+        for image in chart_info['required_images']:
+            image_name = image.split('/')[-1]
+            image_package_name = package['name'] + '-images'
+            image_package = ([p for p in release['packages'] if p['name'] == image_package_name] + [ None ])[0]
```

Comment:
`[p for p in release['packages'] if p['name'] == image_package_name]` will return either a list with elements or the empty list `[]`. We should use that convention to be more pythonic

Reply:
One could change the line below to: 
```
if image_package:
    image_package = image_package[0]
else:
   ....
```